srcpkgs/gnuradio/template

```diff
@@ -1,28 +1,40 @@
 # Template file for 'gnuradio'
 pkgname=gnuradio
-version=3.8.0.0
-revision=9
+version=3.10.0.0
+revision=1
 build_style=cmake
 conf_files="/etc/gnuradio/conf.d/*"
-configure_args="-DENABLE_INTERNAL_VOLK=OFF -DGR_PYTHON_DIR=/${py3_sitelib}"
-hostmakedepends="pkg-config doxygen swig python3-Mako python3-Sphinx
- python3-cairo python3-numpy python3-yaml texlive texlive-latexextra texlive-dvi"
-makedepends="SDL-devel boost-devel fftw-devel gsl-devel jack-devel
- libgsm-devel libusb-devel volk-devel uhd-devel gtk+3-devel python3-devel
- python3-gobject-devel log4cpp-devel gmpxx-devel mpir-devel"
-depends="python3-cheetah3 python3-numpy python3-Mako python3-gobject
- python3-yaml"
-checkdepends="python3-scipy"
+configure_args="-DZEROMQ_INCLUDE_DIRS=${XBPS_CROSS_BASE}/usr/include
+ -Dpybind11_DIR=/${py3_sitelib}/pybind11/share/cmake/pybind11"
+# TODO: add python3-pygccxml (not packaged in Void)
```

Comment:
What does this todo mean? Is it just missing functionality?

Reply:
This is used internally to generate Python bindings. In the release tarballs these binding are pre-generated. In any case, it would be nice to have this package in Void.

Here is what the [documentation](https://github.com/gnuradio/gnuradio/blob/master/docs/PYBIND11.md) says:

---

### Generating/maintaining Python bindings using Pybind11

As of the GNU Radio 3.9 release, python bindings are handled using pybind11, which is inherently different than they were in previous releases

#### Dependencies

- pybind11 > 2.4.3 https://pybind11.readthedocs.io/
- pygccxml https://pygccxml.readthedocs.io/en/develop/install.html
  - This is an optional dependency and basic functionality for OOT generation can be performed without pygccxml
  - It is required for automatically generating bindings for most of the GR source tree

...

Each block binding file contains an automatically generated and maintained comment block that informs CMake when the bindings are out of sync with the header file they refer to, and what to do about it